automation_tools/scripts/ga-migration/README.md

```diff
@@ -1,47 +1,74 @@
 # ga-migration
 
+This folder contains the scripts used to migrate a repository from Travis CI to GitHub actions.
+
+The migration pipeline is *idempotent*, meaning you should be able to run it how many times you want, getting the same results.
+
+The `utils.py` file contains the tasks implementations, while `main.py` imports and applies them according to our needs. `gitflow.py` handles the higher level git and github procedures.
+
 ## Install
 
-```console
-$ virtualenv ~/.virtualenvs/ga-migration
-$ source ~/.virtualenvs/ga-migration/bin/activate
-$ pip install -r requirements.txt
+```bash
+# Activate a virtualenv and install pip requirements
+virtualenv ~/.virtualenvs/ga-migration
+source ~/.virtualenvs/ga-migration/bin/activate
+pip install -r requirements.txt
 ```
 
 ## Migration
 
-Apply migration patches on a cloned repository:
+To apply migration patches on a cloned repository, run `python main.py --targetpath=invenio-charts-js`.
+
+E.g.:
 
 ```bash
-# e.g.
-# git clone https://github.com/inveniosoftware/invenio-charts-js
+# cd to the scripts directory
+cd automation-tools/scripts/ga-migration
+# clone the repository to migrate
+git clone https://github.com/inveniosoftware/invenio-charts-js
```

Comment:
Minor: I'm not sure if it's a good practice to clone the repo inside the `ga-migration` folder. Personally, I've been keeping each repo isolated so I was calling the script like `python automation_tools/scripts/ga-migration/main.py --targetpath ../pynpm`

What do you guys think? @diegodelemos @ParthS007

Reply:
It makes sense but IMO it would make the instructions and example commands much more verbose without adding anything, really

Reply:
I don't see how this is so much more verbose:
```console
$ git clone https://github.com/inveniosoftware/invenio-charts-js /some/dir/invenio-charts-js
$ python main.py --targetpath=/some/dir/invenio-charts-js
```

Actually, it is something that:
1. Is safer
2. It takes fewer assumptions, it is more explicit
3. It is easier to script, if needed be

Reply:
Okay, it makes sense